visma/gui/steps.py

```diff
@@ -32,10 +32,12 @@ def showSteps(workspace):
     Arguments:
         workspace {QtWidgets.QWidget} -- main layout
     """
-    workspace.stepsfigure.suptitle(workspace.output, x=0.35, y=0.98,
+    workspace.stepsfigure.suptitle(workspace.output, y=0.98,
                                    horizontalalignment='center',
                                    verticalalignment='top', size=qApp.font().pointSize()*workspace.stepsFontSize)
     workspace.stepscanvas.draw()
+    hbar = workspace.scroll.horizontalScrollBar()
+    hbar.setValue((hbar.minimum()+hbar.maximum())/2)
```

Comment:
Hey @8hantanu , is this alright?
this will always keep the horizontal scroll bar fixed at the center by default when rendering steps.
Also, this keeps aligned with the *workspace.stepsfigure.suptitle*